### DISCUSSION TITLE
Heroku + Nodejs + Platform Events

### DISCUSSION THREAD

**Discussion by anupamabalahara** [jsforce/jsforce#1471] [Q&A]
Hello

Seeing an issue when trying to use jsforce to subscribe to a platform event.

A platform event gets published using a process builder. I know for sure that it is published correctly, as there is a Chatter Post added to the process builder flow. (just so that I am sure when the publish happened)

I have the below set of code in an express heroku node js app:
```
let conn = new jsforce.Connection({
			loginUrl : <>
			clientId : <>
			clientSecret : <>
			redirectUri : <>
	});
	
	conn.login(process.env.SF_USERNAME, process.env.SF_PASSWORD, function(err, res) {
                                          if (err) { 
						console.error('loginToSF - unable to login to sfdc' + err);
						throw err;
					} else {
                                 console.log('SUBSCRIBING');
                                 conn.streaming.topic('/event/Username_Change_Incident__e').subscribe( function( message ) {
						console.log( 'RECEIVED EVENT' );
					});
}
```
I see the SUBSCRIBING log appear. I see the chatter post appear.
But I never see the RECEIVED EVENT appear.

What am I doing wrong ?

**Comment by sharmavishal94** [comment]
https://github.com/douglascayers/sfdc-swag-level-platform-events-demo/blob/master/app.js

**Comment by kalpeshvyas** [comment]
What is the way we can send this information to front page instead of console.log
